soc/microchip/pic32c/pic32cm_jh/Kconfig

```diff
@@ -3,7 +3,7 @@
 
 config SOC_FAMILY_MICROCHIP_PIC32CM_JH
 	select ARM
-	select PIC32C
+	select MICROCHIP_PIC32C
```

Comment:
This change should be atomic together with the `modules/Kconfig.microchip` because it breaks the build.
My suggestion is that you create a commit after the west changes (because you may need to adapt the hal - not sure) with one commit renaming the PIC32C -> MICROCHIP_PIC32C. Then later, you do the normal flow.
This avoids ping pong commits and the rename to PIC32C -> MICROCHIP_PIC32C can be undestood as a preparation to the next commits that follows.